### PR TITLE
Issue 362: [documentation] update command line tools

### PR DESCRIPTION
**Pull request by zhaijack**
Descriptions of the changes in this PR:

enrich doc for bookie shell

---
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

- [X] Make sure the PR title is formatted like:
    `<Issue # or BOOKKEEPER-#>: Description of pull request`
    `e.g. Issue 123: Description ...`
    `e.g. BOOKKEEPER-1234: Description ...`
- [ ] Make sure tests pass via `mvn clean apache-rat:check install findbugs:check`.
- [X] Replace `<Issue # or BOOKKEEPER-#>` in the title with the actual Issue/JIRA number.

---

### CODE REVIEW

File: site/_data/cli/shell.yaml [hunk 1]
```diff
@@ -1,67 +1,157 @@
...
+  description: List the files in JournalDirectory/LedgerDirectories/IndexDirectories.
+  options:
+  - flag: -journal
+    description: Print list of Journal Files
+  - flag: -entrylog
+    description: Print list of EntryLog Files
+  - flag: -index
+    description: Print list of Index files
 - name: listledgers
+  description: List all ledgers on the cluster (this may take a long time).
```
Comment (lucperkins) [inline site/_data/cli/shell.yaml]: in the cluster

File: site/_data/cli/shell.yaml [hunk 2]
```diff
@@ -1,67 +1,157 @@
...
   options:
   - flag: -meta
+    description: Print metadata
 - name: listunderreplicated
+  description: List ledgers marked as underreplicated, with optional options to specify missingreplica (BookieId) and to exclude missingreplica.
```
Comment (lucperkins) [inline site/_data/cli/shell.yaml]: "missingreplica" should be two words (missing replica)

Comment (lucperkins) [inline site/_data/cli/shell.yaml]: Also: "the missing replica"

**Comment by zhaijack** [comment]
Thanks @lucperkins, updated following your comments.

**Pull request closed without merging**